dissect/target/helpers/fsutil.py

```diff
@@ -561,6 +562,29 @@ def open_decompress(
     return file
 
 
+def reverse_read(fh: io.BytesIO, chunk_size: int = 1024 * 10, reverse_chunk: bool = True) -> Iterator[bytes]:
```

Comment:
Why 10KB and not `io.DEFAULT_BUFFER_SIZE`, which is 8KB and probably better aligned with various OS level caches?